### PR TITLE
[de] translation

### PR DESCRIPTION
**Pull request by sotten**
all missing de words translated

### CODE REVIEW

File: locales/de/de.json [hunk 1]
```diff
@@ -159,12 +159,12 @@
     "Cuba": "Kuba",
     "Curaçao": "Curaçao",
     "Current Password": "Derzeitiges Passwort",
-    "Current Subscription Plan": "Current Subscription Plan",
-    "Currently Subscribed": "Currently Subscribed",
+    "Current Subscription Plan": " Aktueller Abonnementplan",
```
Comment (caouecs) [inline locales/de/de.json:162]: space

**Review by caouecs — CHANGES_REQUESTED** [review]
Just a problem of space

**Comment by caouecs** [comment]
Danke, I fix the problem of space

**Comment by sotten** [comment]
> Danke, I fix the problem of space

Merci

**Pull request merged**